tools/scxctl/src/scx_loader.rs

```diff
@@ -0,0 +1,93 @@
+use clap::ValueEnum;
+use zbus::proxy;
+
+/*
+ * D-bus interface info
+ */
+
+#[proxy(
```

Comment:
why not just use `scx_loader` crate, I don't think code duplication is necessary here. Unless you want to minimize dependencies coming from scx_loader

Reply:
Initially it was to minimize dependencies, but now that scxctl is becoming part of the same repo it would probably make more sense to use the crate

Reply:
After taking a look, its not as simple as just switching to the crate. The zbus interface in `scx_loader `expects sched name to be passed as `SupportedSched` and modes as `SchedMode`, whereas I've been handling them as String and u32. I still think it makes sense to switch to the crate, but would probably be more appropriate as a follow up PR

Reply:
I can send PR which implements that, after merge